src/LayerPlan.cpp

```diff
@@ -1212,7 +1233,15 @@ void LayerPlan::addLinesMonotonic
         {
             continue;
         }
-        addTravel(p0);
+        if(vSize2(current_last_position - p0) < line_width_2)
+        {
+            // Instead of doing a small travel that is shorter than the line width (which is generally done at pretty high jerk & move) do a
+            // "fake" extrusion move
+            addExtrusionMove(p0, config, space_fill_type, 0, false, 1.0, fan_speed);
+        } else
```

Comment:
Code style: `else` on a new line.
```suggestion
        }
        else
```